implementation/src/main/java/io/smallrye/mutiny/groups/UniOnSubscribe.java

```diff
@@ -22,9 +22,9 @@
  *
  * <pre>
```

Comment:
should we rename this class to UniOnSubscription?

Reply:
That'd break runtime compatibility.